feat: introduction section updates (#255)
worldcoin/world-id-docs: src/pages/concepts.mdx

import { Link } from '@/components/Link'

# Core Concepts

> World ID is designed to be as simple to integrate as possible, but there are a few core concepts that are helpful to understand before you start.

Reviewer: ```suggestion
World ID is built for easy integration, but here are a few core concepts that are helpful to understand before you start.
```

## Proof of Personhood

"Proof of Personhood" is distinct from many existing identity systems. It is a way to prove that a person is unique, without revealing any personal information. World ID is not KYC, and it does not disclose your identity or previous actions to anyone.

> World ID supports multiple "Verification Levels" to prove that a person is a unique human. These exist on a spectrum of humanness assurance, availability, and fraud-resistance.

Reviewer: ```suggestion
World ID supports multiple "Verification Levels" to prove that a person is a unique human. These levels provide different degrees of humanness assurance, accessibility, and protection against fraud.
```

<table>
  <thead>
    <tr>
      <th>Verification Level</th>
      <th>Humanness Level</th>
      <th>Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td className="flex items-center"><img src="/icons/credential-phone.svg" alt="" className="h-8 w-8 m-0 mr-2" />Device</td>
      <td className="align-middle">Medium</td>
      <td className="align-middle">Unique mobile device check.</td>
    </tr>
    <tr>
      <td className="align-middle"><img src="/icons/credential-orb.svg" alt="" className="h-8 w-8 m-0 mr-2 inline" />Orb</td>
      <td className="align-middle">Strong</td>
      <td className="align-middle">Biometric verification, uniqueness through iris (<Link href="https://worldcoin.org/blog/engineering/opening-orb-look-inside-worldcoin-biometric-imaging-device">details</Link>).</td>
    </tr>
    <tr>
      <td className="flex items-center"><img src="/icons/credential-face.svg" alt="" className="h-8 w-8 m-0 mr-2" />Orb+</td>
      <td className="align-middle">Very Strong</td>

>       <td className="align-middle">Verify that the same person who visited the Orb is performing a specific action.</td>

Reviewer 2: how about something like "Orb verification, plus authentication which verifies the person using World ID is the legitimate holder"
